### ISSUE TITLE
Combat Timer

### ISSUE THREAD

**Issue by allysson-david** [Excrulon/Tree-of-Savior-Addons#190]
Combat Timer doesn't store the position of widget, it resets every time we change maps.
Sadly, it just so happen to be directly under the fourth quickslot bar...

**Comment by Excrulon** [comment]
yeah, was going to have it save the position but I'm not really working on this anymore. it causes you to get disconnected a lot on map/channel change due to requesting the combat timer too much and there's not really a good way around it. It was more of a science experiment... I suggest just uninstalling it for now.
